apps/web/components/shared/card-variant.tsx

```
@@ -0,0 +1,384 @@
+
+            {!showImpact && (
+                <div className="max-w-md p-6 bg-white rounded-lg shadow">
+                    <h3 className="text-xl font-semibold mb-4">Project Creator</h3>
+                    <div className="flex items-center gap-3 mb-4">
+                        <div className="w-10 h-10 bg-gray-100 rounded-full flex items-center justify-center">
+                            <svg
+                                viewBox="0 0 24 24"
+                                fill="none"
+                                className="w-6 h-6 text-gray-400"
+                            >
+                                <path
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)